go/cmd/dolt/commands/sql.go

```diff
@@ -584,6 +584,8 @@ func GetResultFormat(format string) (engine.PrintResultFormat, errhand.VerboseEr
 		return engine.FormatNull, nil
 	case "vertical":
 		return engine.FormatVertical, nil
+	case "pq":
```

Comment:
I think we should use the format naming as `parquet` instead of `pq`, since some of formats we support here such as `csv` and `json` are also used as file extensions.

Reply:
Good feedback! I thought I had seen another spot where we had used `pq`, but now that I look again, I see `parquet` used for `dolt dump` and `dolt table export` uses the file extension to detect the output format. Thanks for catching this; I'll push a new commit with the format named changed to `parquet`.